doc/classes/SpinBox.xml

```diff
@@ -13,23 +23,33 @@
 			<return type="LineEdit">
 			</return>
 			<description>
+				Returns the [LineEdit] id of the [SpinBox].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="align" type="int" setter="set_align" getter="get_align" enum="LineEdit.Align">
+			[code]set_align[/code] Determines the text alignment of the [SpinBox].
```

Comment:
Setter and getter methods shouldn't be spelled out, as they are created automatically. Moreover, the "idiomatic" way to use properties in GDScript as of Godot 3.0 is to access them directly.